kolibri/content/migrations/0001_initial.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
```

Comment:
@MCGallaspy I think "reset" is the right word here... squashing maintains old migration steps and generates a unification migration.